layouts/partials/head.html

```diff
@@ -6,7 +6,12 @@
   {{ $title_plain := .Title | markdownify | plainify }}
   <title>{{ $title_plain }}</title>
   <meta name="author" content="{{ .Param "author" }}" />
-  <meta name="keywords" content="{{ with .Keywords }}{{ delimit . ", " }}{{ else }}{{ delimit .Site.Params.defaultKeywords ", " }}{{ end }}">
+  {{ $keywords := .Site.Params.defaultKeywords }}
+  {{ if isset .Params "tags" }}{{ range .Params.tags }}{{ $keywords = $keywords | append . | uniq }}{{ end }}{{ end }}
+  {{ if isset .Params "keywords" }}{{ range .Params.keywords }}{{ $keywords = $keywords | append . | uniq }}{{ end }}{{ end }}
+  {{ if gt (len $keywords) 0 }}
+  <meta name="keywords" content="{{ delimit $keywords ", " }}">
```

Comment:
I think can be great to do just one time uniq. Here on L13 (also we will consider to make uniq on line L9)

Reply:
> I think can be great to do just one time uniq. Here on L13

You're right, this is more elegant. I've updated the PR accordingly.

> (also we will consider to make uniq on line L9)

This would be redundant.